app/Models/User.php

```diff
@@ -639,6 +639,11 @@ public function isRegistered()
         return $this->isGroup(UserGroup::GROUPS['default']);
     }
 
+    public function isBot()
+    {
+        return $this->isGroup(UserGroup::GROUPS['bot']);
```

Comment:
Because some users in bot group are actual players as well, change this to check `$this->group_id` instead.